tests/lib/ui/generator.ts

```diff
@@ -0,0 +1,150 @@
+import * as proxyquire from 'proxyquire';
+import * as sinon from 'sinon';
+import test from 'ava';
+
+const inquirer = { prompt() { } };
+const resourceLoader = {
+    getCoreCollectors() { },
+    getCoreFormatters() { },
+    getCoreRules() { },
+    loadRules() { }
+};
+
+const stubPifyMethodObject = { pifyMethod() { } };
+
+const stubPifyObject = {
```

Comment:
Since we don't use `pify` any more. I think we should rename this `stubUtilObject`.